wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -121,7 +122,11 @@ instance Migrate V0.CCoin (V1 Core.Coin) where
 instance Migrate (V1 Core.Coin) V0.CCoin where
     eitherMigrate (V1 c) = pure (V0.encodeCType c)
 
---
+instance (n ~ m, n ~ 12)
+    => Migrate (Mnemonic n) (V0.CBackupPhrase m) where
+    eitherMigrate =
+        Right . V0.CBackupPhrase
```

Comment:
Now seems overkill compare to `Migrate (Mnemonic 12)` :joy:

Reply:
:joy: I'll can it